ldap/servers/slapd/pw.c

```diff
@@ -280,7 +280,8 @@ pw_name2scheme(char *name)
         } else {
             /* if not, let's setup pbkdf2 */
 #ifdef RUST_ENABLE
-            char *pbkdf = "PBKDF2-SHA512";
```

Comment:
I'm not sure it'll be enough, as the default pwd scheme change involves multiple places...

I think we should revert these two commits (but we can combine them into one, of course): https://github.com/389ds/389-ds-base/commit/476df9944f662ff6a834d9817adddb0155916333, https://github.com/389ds/389-ds-base/commit/78d936f55afab6c4e818aa45760ec5551c936a34

Reply:
The default pwhash scheme only comes from one place I thought? Just because it's setup as a hasher, doesn't mean it's the default.

Reply:
I think CI tests also need to be updated.  I'll look into all of this tomorrow...